src/commands/deploy.ts

```diff
@@ -36,6 +34,9 @@ export async function deploy(context: IDeployWizardContext, confirmDeployment: b
     if (target instanceof Uri) {
         context.fsPath = target.fsPath;
         context.telemetry.properties.deploymentEntryPoint = 'fileExplorerContextMenu';
+    } else if (target && !(target instanceof WebAppTreeItem)) {
```

Comment:
Is there a better way to do this?  Since Site is an interface, its instance can't be determined at runtime.  I tried using BaseResource (it's base class) but it wasn't recognizing it as that type.